### ISSUE TITLE
Separating code and phone number

### ISSUE THREAD

**Issue by segux** [nikhiln/ngx-international-phone-number#13]
Can you make a change to separate code from phone number? It will be a better feature for your component.

**Comment by choopage** [comment]
It's not difficult. You can do a PR. I am not the owner of this repo. But I want to say it is not difficult.

**Comment by segux** [comment]
Ok, ill make a pr, ill put a new @Input for countryCode

**Comment by nikhiln** [comment]
@segux I have updated package with latest fixes and support for latest angular version, you might want to update your forked as well.

**Comment by samjacob95** [comment]
Was this issue resolved?

**Comment by segux** [comment]
> @segux I have updated package with latest fixes and support for latest angular version, you might want to update your forked as well.

Sorry for late reply, i was too busy for this fork, i hope this month will be more relaxed to do it.

**Comment by tibinvpaul** [comment]
Hey @segux - were you able to add this feature? I am also looking for this! Thanks.

**Comment by segux** [comment]
No, right now im using the https://github.com/google/libphonenumber.

He has PhoneNumber class with some methods to separate phone numbers and get country code and many other things.

**Comment by tibinvpaul** [comment]
Thanks, @segux.

**Comment by choopage** [comment]
@segux What do you mean we can import phonenumber class with some methods to separate phone numbers? Do you have a gist?

**Comment by segux** [comment]
> @segux What do you mean we can import phonenumber class with some methods to separate phone numbers? Do you have a gist?

Here i did a stackblitz to show you a basic example 
https://stackblitz.com/edit/google-libphonenumber-example?file=src/app/app.component.ts

**Comment by choopage** [comment]
Thanks @segux
